packages/lesswrong/components/recentDiscussion/RecentDiscussionThread.tsx

```diff
@@ -87,11 +87,13 @@ const styles = (theme: ThemeType): JssStyles => ({
     }
   },
   post: {
+    backgroundColor: theme.palette.panelBackground.recentDiscussionThread,
     paddingTop: isFriendlyUI ? 12 : 18,
     paddingLeft: 16,
     paddingRight: 16,
     borderRadius: theme.borderRadius[isFriendlyUI ? "default" : "small"],
     marginBottom: 4,
+    border: '1px solid rgba(72, 94, 144, 0.16)',
```

Comment:
I think we disallow hardcoding rgb values inline; it'll need to be defined in our theme first